spec/player_factory_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'human_player'
+require 'computer_player'
+require 'player_factory'
+require 'display'
+
+RSpec.describe PlayerFactory do
```

Comment:
A general design / naming question - it sounds (to me) a bit bizarre that PlayerValidator has a PlayerFactory, not the other way around. What do you think about it?

Reply:
That's a good point. I'll look into this, thanks!